README.md

```diff
@@ -49,9 +49,9 @@ for path in ~/.config/complgen/*.usage; do
     eval "
 _complgen_jit_$stem () {
     local words cword
-    _get_comp_words_by_ref -n = words cword
-    local prefix="\${COMP_WORDS[\$COMP_CWORD]}"
-    local -a completions=(\$(complgen complete \"$HOME/.config/complgen/${stem}.usage\" bash --prefix="\$prefix" -- \${COMP_WORDS[@]:1:\$COMP_CWORD-1}))
+    _get_comp_words_by_ref -n =: words cword
+    local prefix="\${words[\$cword]}"
+    local -a completions=(\$(complgen complete \"$HOME/.config/complgen/${stem}.usage\" bash --prefix="\$prefix" -- \${words[@]:1:\$cword-1}))
```

Comment:
I have added this already on master before I noticed, so it can be removed (sorry 😬 )